test/language/statements/for-of/iterator-next-reference.js

```diff
@@ -3,39 +3,33 @@
 /*---
 es6id: 13.6.4.13 S5.c
```

Comment:
can you replace this with the current esid? This test doesn't reflect the ES6 specs anymore.

Reply:
✔️